### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by calebeby**
# Testing

- Install
  ```
  npm ci
- Set it up so that your local `fluidlint` version is available globally:
  ```
  npm link
  ```
- Create a liquid file that is invalid. Confirm that running `fluidlint` prints the errors

### CODE REVIEW

File: src/formatter.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
+'use strict';
+
+// This file is copied from eslint-formatter-pretty except that it also outputs
+// the source line that failed the rules because we don't keep track of line/col numbers
+
+const path = require('path');
+const chalk = require('chalk');
```
Comment (tylersticka) [inline src/formatter.js]: Did you consider [`ansi-colors`](https://www.npmjs.com/package/ansi-colors) instead? I reached for `chalk` in another experiment and saw that `ansi-colors` had no dependencies.

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/dist
```
Comment (tylersticka) [inline .eslintignore]: Intentionally missing newline?

File: .prettierignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/dist
```
Comment (tylersticka) [inline .prettierignore]: Intentionally missing newline?

File: .npmrc [hunk 4]
```diff
@@ -0,0 +1 @@
+git-tag-version=false
```
Comment (tylersticka) [inline .npmrc:3]: Curiosity: Why?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+```json
+  "scripts": {
+    "fluidlint": "fluidlint",
+  },
+```
+
+```
+npm run fluidlint
+```
```
Comment (tylersticka) [inline README.md:29]: Will it be possible to specify input files? For example, in our project we only want to scrutinize specific files, elsewhere compatibility isn't a concern.

**Review by tylersticka — COMMENTED** [review]
Works well locally, once I figured out that I had to run `fluidlint` in a directory and not with an input path! (For some reason I was trying to use it the same way I would the `eslint` or `stylelint` CLIs.)

Some comments and questions inline, but I defer final review to @gerardo-rodriguez.

File: .eslintignore (hunk 2 above)
Comment (gerardo-rodriguez) [inline .eslintignore]: Missing new line?

File: .prettierignore (hunk 3 above)
Comment (gerardo-rodriguez) [inline .prettierignore]: Missing new line?

File: src/formatter.js (hunk 1 above)
Comment (calebeby) [reply src/formatter.js]: Thanks for reminding me 👍

File: .npmrc (hunk 4 above)
Comment (calebeby) [reply .npmrc:3]: Added comment

File: README.md (hunk 5 above)
Comment (calebeby) [reply README.md:29]: ✔️

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+      };
+    }
+  }
+];
+
+const main = async () => {
+  const args = process.argv.slice(2);
+  const userGlobs = args.filter(arg => !arg.startsWith('-'));
+  const files = await glob(userGlobs.length > 0 ? userGlobs : ['**/*.liquid'], {
+    ignore: ['node_modules/**/*']
```
Comment (gerardo-rodriguez) [inline src/index.js]: I do not know if this matters but I accidentally ran `fluidlint` at a higher level directory than I meant to run it in. It ended up checking a file that exists within the `node_modules` directory. Is that the expected result?

```
$ fluidlint

  projects/baptist-health/repos/baptist-health-patterns/node_modules/@11ty/eleventy/test/stubs/includer.liquid
  ✖  Expected path to be quoted                                 paths-should-be-quoted
     {% include included %}
```

Comment (calebeby) [reply src/index.js]: Good catch 👍

**Review by gerardo-rodriguez — APPROVED** [review]
LGTM AFAICT 👍

**Pull request merged**